oidc-fapi-conformance-tests/test_runner_fapi.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash +x
+
+CONFORMANCE_SUITE_PATH=./conformance-suite
+PATH_TO_SCRIPTS=./product-is/oidc-fapi-conformance-tests
+IS_SUCCESSFUL=false
+
+#  sed -i '/^.*all_test_modules\ =.*/a \ \ \ \ print("All available OIDC test modules:")\n\ \ \ \ print("==============================================")\n\ \ \ \ print(sorted(all_test_modules.keys()))' $CONFORMANCE_SUITE_PATH/scripts/run-test-plan.py
+
+echo "========================"
+echo "Running Tests"
+echo "========================"
+
+echo
+echo "FAPI Basic test plan - private_key_jwt, plain_fapi, plain_response, by_value"
+echo "-----------------------------"
+echo
+python3 $CONFORMANCE_SUITE_PATH/scripts/run-test-plan.py fapi1-advanced-final-test-plan[client_auth_type=private_key_jwt][fapi_profile=plain_fapi][fapi_response_mode=plain_response][fapi_auth_request_method=by_value] $PATH_TO_SCRIPTS/config/IS_config_fapi_pvtkeyjwt.json 2>&1 | tee fapi-testplan-pvtkeyjwt-log.txt
+echo
+
+echo
+echo "FAPI Basic test plan - mtls, plain_fapi, plain_response, by_value"
+echo "-----------------------------"
+echo
+python3 $CONFORMANCE_SUITE_PATH/scripts/run-test-plan.py fapi1-advanced-final-test-plan[client_auth_type=mtls][fapi_profile=plain_fapi][fapi_response_mode=plain_response][fapi_auth_request_method=by_value] $PATH_TO_SCRIPTS/config/IS_config_fapi_mtls.json 2>&1 | tee fapi-testplan-mtls-log.txt
+echo
+
+# echo
```

Comment:
Why this is commented?

Reply:
Currently the PAR and JARM FAPI profiles are not passing, and hence I have added the lines and commented so we just need to uncomment those 2 lines when those profiles are ready.

JARM issue: https://github.com/wso2/product-is/issues/18062